drag-sort-listview-gradle/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.library'
+
+// This file is here (rather than in the drag-sort-listview project)
+// so we can control how it builds without having to submit pull requests.
+
+dependencies {
+    compile 'com.android.support:support-v4:19.1.0'
+}
+
+android {
+    compileSdkVersion 7
```

Comment:
Ok, cool.

Main reason I ask is because I'm having a hard time even finding API 7 to download. I changed it to latest SDK and everything seems fine ¯\_(ツ)_/¯